src/main/java/com/jcabi/ssh/SSH.java

```diff
@@ -76,34 +76,15 @@
  * @see <a href="http://www.yegor256.com/2014/09/02/java-ssh-client.html">article by Yegor Bugayenko</a>
  */
 @ToString
-@EqualsAndHashCode(of = { "addr", "port", "login", "key" })
+@EqualsAndHashCode(callSuper=false)
```

Comment:
@kpbird here I believe you broke equals by removing the key attribute from it...